x/staking/keeper/invariants.go

```diff
@@ -52,9 +52,8 @@ func AllInvariants(k Keeper, f types.FeeCollectionKeeper,
 	}
 }
 
-// SupplyInvariants checks that the total supply reflects all held not-bonded tokens, bonded tokens, and unbonding delegations
-// nolint: unparam
-func SupplyInvariants(k Keeper, f types.FeeCollectionKeeper,
+// BondedTokensInvariants checks that the total supply reflects all held not-bonded tokens, bonded tokens, and unbonding delegations
```

Comment:
This godoc doesn't match the function name. Are we executing bonded token invariants only or total supply?